M2/Macaulay2/packages/Permutations.m2

```diff
@@ -0,0 +1,98 @@
+newPackage(
+    "Permutations",
+    AuxiliaryFiles => true,
+    Version => "1.0", 
+    Date => "June 4, 2024",
+    Keywords => {"Combinatorics"},
+    Authors => {
+        {Name => "Sean Grate", 
+         Email => "sean.grate@auburn.edu", 
+         HomePage => "https://seangrate.com/"}
+  },
+    Headline => "functions for working with permutations",
+    PackageExports => {
+        -- "SimplicialComplexes",
+        -- "SimplicialDecomposability",
+        -- "Posets",
+        -- "MinimalPrimes"
+    },
+    DebuggingMode => true
```

Comment:
`DebuggingMode` should either be set to `false`, or this option removed completely (since `false` is the default).

Reply:
This should be corrected in some new commits.